### PR TITLE
immer example for createSliceWithImmer

### PR DESCRIPTION
**Pull request by Ebubekir-Tas**
Example case for `createSlicesWithImmer`

Also primitives values for slices can be used with explicit returns that do not modify the draft state as shown in `textSlice` so that's nice

### CODE REVIEW

File: examples/04_immer/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "example",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "dependencies": {
```
Comment (dai-shi) [inline examples/04_immer/package.json:6]: We need `"immer"` too.

**Review by dai-shi — CHANGES_REQUESTED** [review]
Thanks for working on it.

Can you add entries in package.json and README.md too?
Find how `03_actions` is configured there.

File: examples/04_immer/package.json (hunk 1 above)
Comment (Ebubekir-Tas) [reply examples/04_immer/package.json:6]: I added immer as a dependency in the examples folder, I need to also add

```
    "examples:01_counter": "DIR=01_counter vite",
    "examples:02_async": "DIR=02_async vite",
    "examples:03_actions": "DIR=03_actions vite",
    "examples:04_immer": "DIR=04_immer vite"
```

to the root package.json but it seems something desynced my local branch to this pull request and I'm hesitant to make any force pushes

**Comment by dai-shi** [comment]
> I also see stackblitz URLs in the README

It will run when we release a new version. But, I can update README later.

> Noteworthy that to run it in windows the script is

I'm wondering if this works for windows users: https://pnpm.io/cli/run#shell-emulator

**Review by dai-shi — APPROVED** [review]
LGTM!

**Pull request merged**